packages/grpc-js/src/server-call.ts

```diff
@@ -18,13 +18,14 @@
 import { EventEmitter } from 'events';
 import * as http2 from 'http2';
 import { Duplex, Readable, Writable } from 'stream';
+import * as util from 'util';
```

Comment:
`util.isNumber()` and `util.isString()` are deprecated in favor of `typeof value === 'number'` and `typeof value === 'string'`, respectively.